app/assets/stylesheets/helpers/_task-list-title.scss

```diff
@@ -0,0 +1,18 @@
+.task-list-title {
+  @include bold-24;
+
+  margin-bottom: 0;
+  margin-top: 30px;
+
+  @include media(tablet) {
+    margin-top: 180px;
```

Comment:
This looks like a magic number, is this robust against changing content?

Reply:
It's a magic number. I know. The idea is that the task list in the sidebar shouldn't be right at the top, because people tend to ignore it, but if it's a little bit lower, it works better. Unfortunately as you point out, every page has a different layout so it's hard to position this sensibly.

Reply:
Should we be reorganising the grid so that we don't make this assumption?

Reply:
Wouldn't that involve restructuring a ton of page templates?